service/src/main/resources/HTMLTemplate.html

```diff
@@ -1359,6 +1364,183 @@ <h3 class="heading heading--two">Respondent {{counter}}</h3>
     {% set counter = counter + 1 %}
     {% endfor %}
   {% endif %}
+  {% endif %}
+
```

Comment:
Fix indentation in this file please, it will make the upcoming Docmosis epic easier if good spacing persists.

Reply:
there is far too much that needs fixing in the HTML file and I do not want large file changes that are unrelated to this story. I have indented the lines you've mentioned, but will not do any more